### PR TITLE
use native profile only in dev profile

### LINKED ISSUE #179: version 3.1.1 uses native in prod profile
##### **Overview of the issue**

When running the JHipster Registry in the `prod` profile as suggested in the documentation http://www.jhipster.tech/jhipster-registry/
```
./jhipster-registry-3.1.1.war` --spring.profiles.active=prod --security.user.password=admin --jhipster.security.authentication.jwt.secret=secret-key --spring.cloud.config.server.git.uri=https://github.com/jhipster/jhipster-registry-sample-config
```
then JHipster Registry is using Spring Cloud Config `native` profile:
```
        Profile(s):     [native, prod]
----------------------------------------------------------
```

##### **Motivation for or Use Case**

In prod profile I want to be able to use configuration from git. JHipster Registry documentation http://www.jhipster.tech/jhipster-registry/ supports this wish in prod profile.

##### **Reproduce the error**

Run
```
./jhipster-registry-3.1.1.war` --spring.profiles.active=prod --security.user.password=admin --jhipster.security.authentication.jwt.secret=secret-key --spring.cloud.config.server.git.uri=https://github.com/jhipster/jhipster-registry-sample-config
```

##### **Suggest a Fix**

Seems that this problem comes from this commit: https://github.com/jhipster/jhipster-registry/commit/55dc94856e1fd265001a5361d8a51bd8518d0211

Maybe create `bootstrap-dev.yml` and put
```
spring:
    profiles:
        include: native
```
into this new file.

##### **JHipster Registry Version(s)**

This problem exists in version 3.1.1.  
This problem doesn't exist in version 3.1.0

##### **Browsers and Operating System**

Tested both in Centos and Windows. Same behaviour.

- [x] Checking this box is mandatory (this is just to show you read everything)

### PR DESCRIPTION
**Pull request by kaidohallik**
- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/jhipster-registry/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-registry/blob/master/CONTRIBUTING.md) are followed

fix #179

### CODE REVIEW

**Comment by PierreBesson** [comment]
I agree with this change, and this is important that the native profile is not enabled when using a git config source.

**Comment by jdubois** [comment]
And you don't put `git` by default in `prod` mode? I need to check

**Comment by kaidohallik** [comment]
@jdubois Information from git was used without specifying `git`.  
@PierreBesson Using `git` in `dev` is problematic with this solution. If this must be possible then maybe drop `native` at all from configurations? Or in bootstrap.yml switch back to:
```
spring:
    profiles:
        active: dev,native
```
and don't add `bootstrap-dev.yml`.  
Then
* `jhipster-registry.war` uses dev with native
* `jhipster-registry.war --spring.profiles.active=dev` uses dev with git
* `jhipster-registry.war --spring.profiles.active=prod` uses prod with git
* `jhipster-registry.war --spring.profiles.active=prod,native` uses prod with native

So basically revert https://github.com/jhipster/jhipster-registry/commit/55dc94856e1fd265001a5361d8a51bd8518d0211

**Comment by jdubois** [comment]
I don't really understand the original goal of modifying this, yes I would revert back the commit - this was working fine and made sense to me - @PierreBesson are you OK with reverting the commit?

**Comment by jdubois** [comment]
I'm not totally sure what those changes do: I'll revert the commit, it has been working for a long time with no issue, I don't want to take risks
